yellowbrick/features/pca.py

```diff
@@ -24,6 +24,7 @@
 from yellowbrick.features.base import MultiFeatureVisualizer
 from yellowbrick.style import palettes
 from yellowbrick.exceptions import YellowbrickValueError
+from yellowbrick.draw import manual_legend
```

Comment:
This file looks like work from a different PR/feature? Would you mind reverting it so we don't include these changes in this PR?

Reply:
@bbengfort - I copied the latest code but it looks like there is a hidden character in the last line.  I am not sure how to fix it.  Do you have any ideas?

Reply:
Add a newline to the end of the file? I think that should fix it.